tests/02_00_02-2wp_segwit_compatible.js

```diff
@@ -1,74 +1,65 @@
         await rskUtils.mineAndSync(rskTxHelpers);
 
-        return federationAddress;
-    });
+        const bridge = getBridge(rskTxHelper.getClient(), await getLatestActiveForkName());
+        const federationAddress = await bridge.methods.getFederationAddress().call();
+        const federationAddressBalanceInitial = Number(await btcTxHelper.getAddressBalance(federationAddress));
+        console.log(`federationAddressBalanceInitial: ${federationAddressBalanceInitial}`)
 
-    it('should work when using p2sh-p2wpkh address', async () => {
-        try {
-            const INITIAL_BTC_BALANCE = bitcoin.btcToSatoshis(40);
-            const INITIAL_RSK_BALANCE = bitcoin.btcToSatoshis(10);
-            const MAX_EXPECTED_FEE = bitcoin.btcToSatoshis(0.001);
+        const recipientRskAddressInfo = getDerivedRSKAddressInformation(senderAddressInfo.privateKey, btcTxHelper.btcConfig.network);
+        await rskTxHelper.importAccount(recipientRskAddressInfo.privateKey);
+        const unlocked = await rskTxHelper.unlockAccount(recipientRskAddressInfo.address);
+        expect(unlocked, 'Account was not unlocked').to.be.true;
 
-            var addresses = await pegClient.generateNewAddress('test', 'p2sh-segwit');
-            expect(addresses.inRSK).to.be.true;
+        await btcTxHelper.fundAddress(senderAddressInfo.address, MINIMUM_PEGIN_VALUE_IN_BTC + btcTxHelper.getFee());
+        
+        const btcPeginTxHash = await sendPegin(rskTxHelper, btcTxHelper, senderAddressInfo, MINIMUM_PEGIN_VALUE_IN_BTC);
+        console.log(`btcPeginTxHash: ${btcPeginTxHash}`)
 
-            await btcClient.sendToAddress(addresses.btc, INITIAL_BTC_BALANCE);
-            await btcClient.generate(1);
-            await test.assertBitcoinBalance(addresses.btc, INITIAL_BTC_BALANCE, "Wrong initial BTC balance");
-            await wait(1000);
+        await ensurePeginIsRegistered(rskTxHelper, btcPeginTxHash);
 
-            var btcBalances = await btcClient.getAddressBalance(federationAddress);
-            initialFederationBalance = btcBalances[federationAddress] || 0;
+        const federationAddressBalanceAfterPegin = Number(await btcTxHelper.getAddressBalance(federationAddress));
+        console.log(`federationAddressBalanceAfterPegin: ${federationAddressBalanceAfterPegin}`)
+        expect(Number(federationAddressBalanceAfterPegin)).to.be.equal(Number(federationAddressBalanceInitial + MINIMUM_PEGIN_VALUE_IN_BTC))
 
-            await test.assertLock(addresses, [{ address: federationAddress, amount: INITIAL_RSK_BALANCE }]);
-        } 
-        catch (err) {
-            throw new CustomError('Transfer BTC to RBTC failure', err);
-        }
+        const senderAddressBalanceAfterPegin = Number(await btcTxHelper.getAddressBalance(senderAddressInfo.address));
+        console.log(`senderAddressBalanceAfterPegin: ${senderAddressBalanceAfterPegin}`)
```

Comment:
Remove all the `console.log` from this file.

Reply:
will do for the final version.